arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1906,13 +1906,13 @@ int ClusterInfo::dropCollectionCoordinator(
   double const endTime = TRI_microtime() + realTimeout;
   double const interval = getPollInterval();
 
-  auto dbServerResult = std::make_shared<int>(-1);
+  auto dbServerResult = std::make_shared<std::atomic<int>>(-1);
   auto errMsg = std::make_shared<std::string>();
```

Comment:
`errMsg` is now effectively unused.

Reply:
True... Removing it and restarting tests.